karma.conf.js

```diff
@@ -20,7 +20,7 @@ module.exports = function (config) {
       'karma-sourcemap-loader'
     ],
 
-    browsers: ['PhantomJS'],
+    browsers: process.env.CI === true ? ['PhantomJS'] : [],
```

Comment:
I believe command line flags override this so keeping this array here allows karma to always default to Phantom.

Reply:
Oh, I missed an important script.

> `yarn test-watch`

This script will watch for file changes and start a server. PhantomJS runs as well but the developer can choose to open a browser to `localhost:9876`

:facepalming:

Reply:
To sum everything up:

`yarn test` here defaults to CI mode whereas it's dev (watch) mode in bigmaven. Ideally, the script flows should be synced with bigmaven so running `yarn test` here runs in dev mode since it's the more frequent option.

Not a blocking change though, but something that would be nice to have.